packages/grafana-ui/src/components/Forms/InlineFieldRow.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, ReactNode, HTMLProps } from 'react';
+import { css, cx } from 'emotion';
+import { useStyles } from '../../themes';
+
+export interface Props extends Omit<HTMLProps<HTMLDivElement>, 'css'> {
```

Comment:
Hmm, why `css` is omitted here? How come is this a property on HTMLProps type?

Reply:
For some reason it ends up in the component's Prop table, which is confusing for the users. Could be related to https://github.com/grafana/grafana/issues/26512 maybe?